test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -570,11 +570,26 @@ Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows"
             $ci[1].Name | Should -MatchExactly $filenamePattern
             $ci[2].Name | Should -MatchExactly $filenamePattern
         }
+        It "Get-ChildItem -Name gets content of linked-to directory" {
+            # The test depends on the files created in previous test:
```

Comment:
Tests should not depend on previous test.  Would be better to use `BeforeEach{}` if the system state needs to be set or if just a few test cases add a helper function.  In this specific case, it may be better to turn the previous test case into `-TestCases`

Reply:
I did want only add new tests for -Name parameter but found current tests is very sensitive. I don't want to break something in the tests in the PR.
I think we could review all tests in the file when we will work on updating new file enumerator. Now I plan only pull new enumerator for `-Name` to keep follow PR as small as possible.

Reply:
Seems like you could do something like:

```powershell
It "Get-ChildItem gets content of linked-to directory: <testCase>" -TestCases @(
  @{ testCase = '-Name not used'; Name = $false }
  @{ testCase = '-Name used'; Name = $true }
){
    params($Name)
    $filenamePattern = "AlphaFile[12]\.txt"
    New-Item -ItemType SymbolicLink -Path $alphaLink -Value $alphaDir -Force > $null
    $args = @{
        Path = $alphaLink
        Name = $Name
    }
    $ci = Get-ChildItem @args
    $ci.Count | Should -Be 3
    $ci[1].Name | Should -MatchExactly $filenamePattern
    $ci[2].Name | Should -MatchExactly $filenamePattern
}
```